### ISSUE TITLE
Safari 14 audio test returns empty values for typical formats

### ISSUE THREAD

**Issue by beshur** [Modernizr/Modernizr#2604]
I am working on a project and noticed that in Safari 14 (technological preview on Mac Catalina) `Modernizr.audio` returns this:

`{ogg: "", mp3: "", opus: "", wav: "", m4a: "maybe"}`

However, it plays MP3 just fine on other sites, most online test suites show it can play MP3 and M4A.

**Comment by Markel** [comment]
Confirmed using Browserstack, and it is definitely something about Safari 14 and not the device. Now, why does it happen, no idea... Nothing in the changelog seems to say anything about audio... It may be anti-fingerprinting technology? Or simply just a bug from Safari 14?

**Comment by Markel** [comment]
Maybe have to do something with the bunch of things related to audio of [release 113??](https://developer.apple.com/safari/technology-preview/release-notes/), but honestly I don't even know if that is the last version. I'm not very familiar with Safari (don't even own a iOS device)

**Comment by beshur** [comment]
Safari 14 is the next stable version (currently in beta), coming out with the next Mac OS version later this year.

While something changed in Safari, [some other site](https://html5test.com) for testing HTML5 features do a different check and yield positive results for mp3.

**Comment by beshur** [comment]
Apple just released Safari 14
https://www.macrumors.com/2020/09/16/apple-releases-safari-14-for-mac/

**Comment by Markel** [comment]
Yeah, we should get to it. But sorry, university is slowing me 😅😅

**Comment by beshur** [comment]
That's going to break a lot of sites, more people inbound, I can assure you.

In the meantime I found that `codecs` is missing from `canPlayType` in Safari.

**Comment by Markel** [comment]
> In the meantime I found that `codecs` is missing from `canPlayType` in Safari.

@beshur What do you mean by that?

**Comment by Markel** [comment]
I mean we cannot delete the codecs part as that wouldn't make sense as the container and the codecs are two different things (something even more apparent in the video test, which is also affected by this bug). Honestly I don't really know what can we do in this case...
